rmgpy/molecule/draw.py

```diff
@@ -49,6 +49,7 @@
 import math
 import os.path
 import re
+import warnings
```

Comment:
does this do anything? I can't see what

Reply:
I had intended for [this line](https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2539/files/5ad2f2484df06d5910f3c09ad94ccf3ccdee9c3d#diff-8571db88680406d006088de6253f48278b66c7b3f6fd15f711878f822333eed3R70) to use `warnings.warn` but since I changed it I will remove this.